src/content/en/2020/markup.md

```diff
@@ -0,0 +1,626 @@
+</figure>
+
+These elements come from three sources: [Yandex Metrics](https://metrica.yandex.com/about) (`ym-`), an analytics solution we’ve also seen last year; [Slider Revolution](https://www.sliderrevolution.com/) (`rs-`), a WordPress slider, for which there are more elements to be found near the top of the sample; and [Wix](https://www.wix.com/) (`wix-`), a website builder.
+
+Other groups that stand out include [AMP markup](https://amp.dev/) with `amp-` elements like `amp-img` (11,700 cases), `amp-analytics` (10,256) and `amp-auto-ads` (7,621), as well as [Angular](https://angular.io/) `app-` elements like `app-root` (16,314), `app-footer` (6,745), and `app-header` (5,274).
+
+### Obsolete elements
+
+There are more questions to ask about the use of HTML, and one may relate to obsolete elements—elements like `applet`, `bgsound`, `blink`, `center`, `font`, `frame`, `isindex`, `marquee`, or `spacer`.
+
+In our mobile data set of 6.3 million pages, around 0.9 million pages (14.01%) contained one or more of these elements. Here are the Top 9, those elements that were used more than 10,000 times:
+
+<figure markdown>
+| Element | Occurrences | Percentage of pages with this element |
+|---|---|---|
+| `center` | 458,402 | 7.22% |
+| `font` | 430,987 | 6.79% |
+| `marquee` | 67,781 | 1.07% |
+| `nobr` | 31,138 | 0.49% |
+| `big` | 27,578 | 0.43% |
+| `frame` | 19,363 | 0.31% |
+| `frameset` | 19,163 | 0.30% |
+| `strike` | 17,438 | 0.27% |
+| `noframes` | 15,016 | 0.24% |
+<figcaption>{{ figure_link(caption="Obsolete elements with more than 10,000 uses.", sheets_gid="1972617631", sql_file="pages_element_count_by_device_and_obsolete_elements.sql") }}</figcaption>
+</figure>
+
+Why are these still alive—even `spacer` is still being used 1,584 times, and present on every 5,000th page. We know that Google has been using a `center` element on [their homepage](https://www.google.com/) [for 22 years](https://web.archive.org/web/19981202230410/https://www.google.com/) now, but does that find so many imitators?
+
+<div class="note">
+
+#### `isindex`
+
+If you were wondering: The [`isindex` element](https://www.w3.org/TR/html401/interact/forms.html#edef-ISINDEX) was present once. (It was part of the HTML specs [until version 4.01, and of XHTML in 1.0](https://meiert.com/en/indices/html-elements/), yet only properly [specified in 2006](https://lists.w3.org/Archives/Public/public-whatwg-archive/2006Feb/0111.html), and [removed in 2016](https://github.com/whatwg/html/pull/1095).)
+
+</div>
+
+### Proprietary and made-up elements
+
+In our set of elements we found some that were neither standard HTML (nor SVG nor MathML) elements, nor custom ones, nor obsolete ones, but somewhat proprietary ones. The Top 10 that we identified were the following:
+
+<figure markdown>
+| Element | Percentage of pages with this element |
```

Comment:
Ditto:

```suggestion
| Element | Percentage |
```

Reply:
Dto. (would prefer to look at this later, with full context)